src/client/templates/cardModal.html

```diff
@@ -0,0 +1,61 @@
+<template name="cardModal">
+  {{#with cardModalInfo this.id}}
+  <div class="modal" id="{{_id}}" tabindex="-1" role="dialog">
+    <div class="modal-dialog" role="document">
+      <div class="modal-content">
+        <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true"><i class="fa fa-times"></i></span></button>
+          <h4 class="modal-title">{{author.profile.name}}</h4>
+        </div>
+        <div class="modal-body">
+          <p>Thought: {{text}}</p>
+          <p>Category: {{category}}</p>
+          <p>Tags: {{#each tags}} {{this}}, {{/each}} </p>
+          <p>Author: {{author.profile.name}}</p>
+          <p>Created at: {{createdAt}}</p>
+          <hr>
+          <form id="commentFormField">
+            <div class="commentFormGroup input-group">
+              <span class="input-group-addon text-success" for="Comment">Comment</span>
+              <input class="form-control" type="text" id="comment" placeholder="Leave a comment.">
+              <span class="input-group-btn"><button class="btn btn-primary" id="submitCommentButton" type="submit">Submit</button></span>
+            </div>
+          </form>
+          <hr>
+          <div>
+            <h4>Comments</h4>
+            <span id="expand"><i class="fa fa-caret-right"></i></span>
+          </div>
+          <ul class="collapsible popout" data-collapsible="accordion">
+            {{#each comments}}
+            <li>
+              <div class="collapsible-header blue-grey darken-4">
+                {{#if this.image}}
+                <img class="img-responsive img-rounded" src="{{this.image}}" /> {{/if}}
+                <p class="white-text">&nbsp;&nbsp; {{this.text}}</p>
+              </div>
+              <div class="collapsible-body">
+                <p>Author: {{this.author}}</p>
+              </div>
+            </li>
+            {{/each}}
+          </ul>
+        </div>
+        <div class="modal-footer">
+          <button id="cardModalClose" type="button" class="btn btn-primary" data-dismiss="modal">Close</button>
+          <button type="button" class="btn btn-danger" id="deleteCardButton" data-dismiss="modal">Delete Card</button>
```

Comment:
I kinda like having the delete button here, as it's less likely to click it by accident. Not sure if we even need to confirm card deletion at this point.